services/OneCampaignMonitor_SubscribersService.php

```diff
@@ -29,4 +39,82 @@ public function add($listId, $email, $name=null, $customFields=array(), $resubsc
         return true;
     }
 
+    /**
+     * Determines is a subscriber exists in a list
+     * @param  $listId
+     * @param  $email
+     * @return  Boolean
+     * @throws Exception
+     */
+    public function exists($listId, $email) {
+        if (!$listId) {
+            throw new Exception('List ID is required');
+        }
+        if (!$email) {
+            throw new Exception('Please provide a valid email address');
```

Comment:
I don't really care, here, because even if it's malformed, the request will still go through. We can rely on Campaign Monitor's error messaging for this.